src/App.tsx

```diff
@@ -1,15 +1,59 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import classNames from 'classnames';
+import debounce from 'lodash.debounce';
 
-export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+interface Props {
+  debounceDelay?: number;
+}
+
+export const App: React.FC<Props> = ({ debounceDelay = 300 }) => {
+  const [query, setQuery] = useState('');
+  const [dropdownOpen, setDropdownOpen] = useState(false);
```

Comment:
Rename all boolean variables using these [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
```